### ISSUE TITLE
somes pdb code that ParmEd failed to load

### ISSUE THREAD

**Issue by hainm** [ParmEd/ParmEd#820]
So I got a bunch of pdb codes from `pdbfixer` and try with ParmEd.

List of codes: 
```python
    pdbcodes_failes = ['1AAF', '1ANP', '1AQQ', '1AQS', '1AW6', 
                       '1BBO', '1BFZ', '1BZB', '1D0W']
    import parmed
    CODES = pdbcodes_failes
    for code in (CODES):
        print(code)
        parmed.download_PDB(code)
```

some errors
```bash
1AAF:  'NoneType' object has no attribute 'name'
1ANP:  'NoneType' object has no attribute 'name'
1AQQ:  'NoneType' object has no attribute 'name'
1AQS:  Atom 518 differs in MODEL 3 [ASN H vs. ASN OXT]
1AW6:  'NoneType' object has no attribute 'name'
1BBO:  'NoneType' object has no attribute 'name'
1BFZ:  Inconsistent atom numbers in some PDB models
1BZB:  Atom 369 differs in MODEL 3 [ARG NH1 vs. ARG NH2]
1D0W:  Atom 98 differs in MODEL 25 [LYS HZ1 vs. LYS HZ3]
```

**Comment by hainm** [comment]
The reason I tried this because `pdb4amber` is using ParmEd to fix pdb now.

**Comment by swails** [comment]
1AQS is broken. Atom 518 is OXT in the third model, but is H in the first. There *is* no OXT atom in the first model as far as I can tell. This violates the PDB standard that states all atoms must exist in the same order in all models for a NMR ensemble. 

All of the PDBs with the NoneType exception are manifestations of the same bug fixed in #841. 1BZB and 1D0W look like broken pdbs given the error message.

**Comment by hainm** [comment]
Great. I Iike ParmEd more.

**Comment by swails** [comment]
This is mostly fixed by #841, and the remaining PDBs that don't work are actually broken :).

Maybe I should add an option to only read a single model to sidestep this error.  But that's another feature.  I'm going to close this.
